src/v2/Components/SavedSearchAlert/types.ts

```diff
@@ -21,27 +21,38 @@ export interface SearchCriteriaAttributes {
   sizes?: string[] | null
 }
 
+export type SearchCriteriaAttributeKeys = keyof SearchCriteriaAttributes
+
+// TODO: Remove it
 export interface SavedSearchEntityArtist {
   id: string
   name: string
   slug: string
 }
 
-interface SavedSearchEntityOwner {
+export interface SavedSearchEntityOwner {
   type: PageOwnerType
   slug: string
   id: string
   name: string
 }
 
+export interface SavedSearchEntityCriteria {
+  displayValue: string
+  value: string
+}
+
+export type SavedSearchDefaultCriteria = Record<
+  string, // TODO: Specify allowed keys
+  SavedSearchEntityCriteria | SavedSearchEntityCriteria[]
```

Comment:
Note that the value for default criteria can be single element or an array of elements. This is necessary in order to extract search criteria correctly when an array of elements is expected (for example, as for `artistIDs` field)

**Single element**
```javascript
defaultCriteria: {
  offerable: {
    displayValue: "Offerable",
    value: "offerable",
  },
}
```

**Multiple elements**
```javascript
defaultCriteria: {
  artistIDs: [
    {
      displayValue: artist.name ?? "",
      value: artist.internalID,
    },
  ],
}
```